client/src/components/core/ms-modal/MsModal.vue

```diff
@@ -0,0 +1,192 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-page>
+    <ion-buttons
+      slot="end"
+      class="closeBtn-container"
+      v-if="closeButtonEnabled"
+    >
+      <ion-button
+        slot="icon-only"
+        @click="cancel()"
+        class="closeBtn"
+      >
+        <ion-icon
+          :icon="close"
+          size="large"
+          class="closeBtn__icon"
+        />
+      </ion-button>
+    </ion-buttons>
+    <div class="ms-modal">
+      <ion-header class="ms-modal-header">
+        <ion-title class="ms-modal-header__title title-h2">
+          {{ title }}
+        </ion-title>
+        <ion-text
+          class="ms-modal-header__text body"
+          v-if="subtitle"
+        >
+          {{ subtitle }}
+        </ion-text>
+      </ion-header>
+      <div
+        class="ms-modal-content inner-content"
+      >
+        <slot />
+      </div>
+      <ion-footer class="ms-modal-footer">
+        <ion-toolbar>
```

Comment:
You need to change existing `--min-height` value because the container must be the same height as the children. 

Possible solution: Add a class and add `--min-height: 0`